src/figanos/matplotlib/_style.py

```diff
@@ -0,0 +1,8 @@
+from pathlib import Path
+
+
+def get_mpl_styles() -> dict[str, Path]:
```

Comment:
there is already a `get_mpl_styles` in `utils`. why a new one here ?

Reply:
I didn't realize that. I was basing this off @sarahclaude's example. Will adjust.